portfolio/.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
In the future, make sure you don't add unrelated files like these to your commit. (This will become more important in group projects, because files like this may cause unnecessary merge conflicts to occur.)